common/openapi.yml

```diff
@@ -17,6 +17,27 @@ paths:
           $ref: '#/components/responses/ServerError'
         '503':
           $ref: '#/components/responses/SystemStatusResponse'
+  "/api/batch/v1/runs":
```

Comment:
Is `batch` needed in API path if we are only submitting one workflow and not batch? Or is this just WES convention?